app/main.py

```diff
@@ -1,3 +1,117 @@
+import json
+
+import datetime
+
+import os
+
+
 def shop_trip():
-    # write your code here
-    pass
+    # with open("config.json", "r") as f:
+    with open("app/config.json", "r") as f:
+        data = json.load(f)
+
+    fuel_price = data["FUEL_PRICE"]
+
+    cust_lst = [data["customers"][i]["name"]
+                for i in range(len(data["customers"]))]
```

Comment:
too complicated, why don't `[customer["name"] for customer in data["customers"]]`